advanced/Scripts/query.sh

```diff
@@ -27,6 +28,8 @@ fi
 # Set this only after sourcing pihole-FTL.conf as the gravity database path may
 # have changed
 gravityDBfile="${GRAVITYDB}"
+gravityOLDfile="$(dirname -- "${gravityDBfile}")/gravity_old.db"
```

Comment:
Why don't you use `${piholeDir}/gravity_old.db`

Reply:
you can change the location of the gravity database in setupVars.conf, see [here](https://docs.pi-hole.net/ftldns/configfile/#file_GRAVITYDB). In order to save the old and the new database in the same location, this code extracts the path from the configured GRAVITYDB variable (if changed, usually not), and adds gravity_old.db to construct the full filename. gravity needs to know where both files are, in order to allow a switch of the database (pihole -g -s)